node/src/node_configurator/configurator.rs

```diff
@@ -1034,9 +1088,9 @@ mod tests {
         );
         let neighborhood_recording = neighborhood_recording_arc.lock().unwrap();
         assert_eq!(
-            neighborhood_recording.get_record::<NewPasswordMessage>(0),
-            &NewPasswordMessage {
-                new_password: "new_password".to_string()
+            neighborhood_recording.get_record::<ConfigurationChangeMessage>(0),
```

Comment:
This should stay the way it was with the old `NewPasswordMessage`.

Reply:
Done 👍